src/main/java/com/coyoapp/tinytask/repository/FileRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.coyoapp.tinytask.repository;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
```

Comment:
the repositories in this package are spring data repository. could be confuse for someone.

Reply:
Creating another repository package will be more confusing. By default repository is a data access layer, in this case, implementation of a repository can be changed to use database storage, sending file to some cloud storage.